linkis-engineconn-plugins/engineconn-plugins/sqoop/src/main/resources/log4j2-engineconn.xml

```diff
@@ -0,0 +1,80 @@
+<!--
```

Comment:
`wds.linkis.engineconn.log4j2.xml.file` default value has been changed.
log4j2-engineonn.xml need be renamed as log4j2.xml

Reply:
ok